### DISCUSSION TITLE
谷歌浏览器访问链接提示ERR_SSL_PROTOCOL_ERROR

### DISCUSSION THREAD

**Discussion by senssic** [natfrp/wiki#189] [使用问题]
之前是好的配置正确且可用，许久未访问，今天访问提示  https://cn-zz-bgp-2.natfrp.cloud:55515/

<img width="997" alt="image" src="https://user-images.githubusercontent.com/3918554/196721185-7bef5bcc-d1be-4af0-ba90-872a30da9bd9.png">

跪求大佬指点迷津

**Comment by Daizy4679** [comment]
我也是，不过换用数字那个IP地址就能正常访问认证，不过我远程也要换成数字IP地址才能正常连接

**Reply by h1gh-noon** [reply]
插眼 解决了踢我一下😪

**Comment by Daizy4679** [comment]
你好，您的邮件我已收到，感谢您的来信！我的地盘我做主！

**Comment by momomobinx** [comment]
2022年11月25日使用域名一样出现 ERR_SSL_PROTOCOL_ERROR 使用ip:port 可以访问

**Reply by h1gh-noon** [reply]
枣庄多线？看站内通知 => 枣庄多线节点屏蔽 TLS 通知

**Reply by momomobinx** [reply]
没注意到😂

**Comment by Daizy4679** [comment]
你好，您的邮件我已收到，感谢您的来信！我的地盘我做主！
